src/components/Resources/ResourcePage.js

```diff
@@ -15,37 +16,27 @@ const useStyles = makeStyles({
 });
 
 function ResourcePage({ matchProps }) {
-  const [resource, setResource] = useState({});
-  const [loading, setLoading] = useState(true);
-
-  useEffect(() => {
-    axios
-      .get('/api/v1/resources/' + matchProps.match.params.guid)
-      .then(function(response) {
-        // handle success
-        setResource(response.data);
-        setLoading(false);
-      })
-      .catch(function(error) {
-        // handle error
-        console.log(error);
-      });
-  }, [matchProps.match.params.guid]);
-
+  const resourceId = matchProps.match.params.guid;
+  // TODO: Handle Error cases
+  const { isLoading, data } = useQuery(['resource', resourceId], getResource);
```

Comment:
Thanks for figuring out this syntax for getting an individual resource ID!

Reply:
I'm not actually sure why we pass down the whole `matchProps` object to the ResourcePage when only need the `guid` here 🤔

Reply:
Yeah, can totally do 

```
            <Route
              path="/resources/:guid"
              render={matchProps => (
                <ResourcePage resourceId={matchProps.match.params.guid} />
              )}
            />
```

and 
```
function ResourcePage({ resourceId }) {
...
ResourcePage.propTypes = {
  resourceId: PropTypes.string,
};
```

if you want :) I don't have a preference for either.